usaspending_api/api_contracts/contracts/v2/federal_accounts/federal_account_totals/program_activities.md

```diff
@@ -0,0 +1,106 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Program Activities Total [/api/v2/federal_accounts/{federal_account_code}/program_activities/total{?limit,page}]
+
+This endpoint returns an array of each program activity's obligation, code, name, and type for the specified federal account 
+
+## POST
+
++ Parameters
+    + `federal_account_code`: `431-0500` (required, string)
+        Federal account code consisting of the AID and main account code
+    + `limit`: 10 (optional, number)
```

Comment:
Since this is a POST with a request body we can keep the values in the json request instead of the query parameters.